partcad/src/partcad/assembly_config_manufacturing.py

```
@@ -0,0 +1,40 @@
+METHOD_NONE = None
+METHOD_ASSEMBLE_PARTCAD_BASIC = 1
+
+_METHOD_MAP: dict[str, int] = {
+    "basic": METHOD_ASSEMBLE_PARTCAD_BASIC,
+}
+
+
+class AssemblyConfigManufacturing:
+    method: int | None
+
+    def __init__(self, final_config) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `final_config`

(ANN001)